spring-cloud-azure-starters/spring-cloud-starter-azure-config/README.md

```diff
@@ -48,6 +48,7 @@ spring.cloud.azure.config.stores[0].name | Name of the configuration store, requ
 spring.cloud.azure.config.stores[0].prefix | The prefix of the key name in the configuration store, e.g., /my-prefix/application/key.name | No |  null
 spring.cloud.azure.config.stores[0].connection-string | Required when `name` is empty, otherwise, can be loaded automatically on Azure Virtual Machine or App Service | Conditional | null
 spring.cloud.azure.config.stores[0].label | Comma separated list of label values, by default will query empty labeled value. If you want to specify *empty*(null) label explicitly, use `%00`, e.g., spring.cloud.azure.config.stores[0].label=%00,v0 | No |  null
+spring.cloud.azure.config.stores[0].watched-key | The single watched key(or key pattern) used to indicate configuration change, has to be configured when the watch feature is enabled.  | Conditional |  null
```

Comment:
I suppose we don't support key pattern and this property is not required to be configured.

Reply:
Missed this and updated.